src/Resources/contao/languages/de/tl_content.php

```diff
@@ -17,6 +17,7 @@
 $GLOBALS['TL_LANG']['tl_content']['sliderButtons'] = ['Steuerschaltflächen','Zeigt Vor- und Zurück-Schaltflächen an.'];
 $GLOBALS['TL_LANG']['tl_content']['sliderPagination'] = ['Nummerierung','Zeigt eine klickbare Nummerierung für die einzelnen Slides an.'];
 $GLOBALS['TL_LANG']['tl_content']['sliderPaginationType'] = ['Art der Nummerierung','Die Art der Nummerierung.'];
+$GLOBALS['TL_LANG']['tl_content']['sliderAutoheight'] = ['Autoheight','Wenn aktiv, passt sich die Höhe des Sliders an die Höhe des aktuell aktiven Elements an.'];
```

Comment:
For the German translation I would call it `Automatische Höhe`.

Reply:
I have changed the translation.